tests/integration/aof-multi-part.tcl

```diff
@@ -1166,5 +1166,138 @@ tags {"external:skip"} {
             set d2 [r debug digest]
             assert {$d1 eq $d2}
         }
+
+        test "AOF will not open incr file until the first AOFRW success when AOF is dynamically enabled" {
+            # Clear all data and delete appenddir to get an empty redis
+            r flushall
+            r config set appendonly no
+            catch {exec rm -rf $aof_dirpath}
+            r debug loadaof
+
+            assert_equal 0 [r dbsize]
+
+            # Increase AOFRW execution time to give us enough time to kill it
+            r config set rdb-key-save-delay 10000000
+
+            # Start write load
+            set load_handle0 [start_write_load $master_host $master_port 10]
```

Comment:
I see 10s is the maximum value in all tests, how much do you think is appropriate for this test?

Reply:
well, maybe it's enough.. considering the timeout for another component in the test is also 10s
i.e.
```tcl
            set maxtries 1000
            set delay 10
```
so if the machine is too slow, the test will fail anyway...

but since we're stopping them manually, i see no reason not to ask for 100s workloads (i agree, the same can be argued about many other tests)